### PR TITLE
Added arm64 documentation.

### PR DESCRIPTION
**Pull request by seriva**

### CODE REVIEW

File: docs/home/ARM.md [hunk 1]
```diff
@@ -0,0 +1,568 @@
+# ARM
+
+From Epiphany 1.1.0 preliminary support for the ```arm64``` architecture was added. As the ```arm64``` architecture is relatively new to the datacenter at the time of writing only a subset of providers, operating system, components and applications are supported. Support will be extended in the future when there is a need for it.
```
Comment (rafzei) [inline docs/home/ARM.md]:
```suggestion
From Epiphany v1.1.0 preliminary support for the ```arm64``` architecture was added. As the ```arm64``` architecture is relatively new to the datacenter at the time of writing only a subset of providers, operating system, components and applications are supported. Support will be extended in the future when there is a need for it.
```

File: docs/home/ARM.md [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+| load_balancer | :heavy_check_mark: | :x: | :x: |
+| postgresql | :heavy_check_mark: | :x: | :x: |
+| ignite | :heavy_check_mark: | :x: | :x: |
+| opendistro_for_elasticsearch | :heavy_check_mark: | :x: | :x: |
+| single_machine | :heavy_check_mark: | :x: | :x: |
+
+***Notes***
+
+- For the ```postgresql``` component the ```pgpool``` and ```pgbouncer``` extensions for load-balancing and replication are not jet supported on ```arm64```. These should be disabled in the ```postgressql``` and ```applications``` configurations.
+- While not defined in any of the component configurations, the ```elasticsearch_curator``` role us currently not supported on ```arm64``` and should be removed from the ```feature-mapping``` configuration if defined.
```
Comment (rafzei) [inline docs/home/ARM.md]:
```suggestion
- While not defined in any of the component configurations, the ```elasticsearch_curator``` role is currently not supported on ```arm64``` and should be removed from the ```feature-mapping``` configuration if defined.
```

**Review by rafzei — CHANGES_REQUESTED** [review]
Looks good but I've added minor changes

**Review by rafzei — DISMISSED** [review]

File: docs/home/ARM.md [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+| monitoring | :heavy_check_mark: | :x: | :x: |
+| load_balancer | :heavy_check_mark: | :x: | :x: |
+| postgresql | :heavy_check_mark: | :x: | :x: |
+| ignite | :heavy_check_mark: | :x: | :x: |
+| opendistro_for_elasticsearch | :heavy_check_mark: | :x: | :x: |
+| single_machine | :heavy_check_mark: | :x: | :x: |
+
+***Notes***
+
+- For the ```postgresql``` component the ```pgpool``` and ```pgbouncer``` extensions for load-balancing and replication are not jet supported on ```arm64```. These should be disabled in the ```postgressql``` and ```applications``` configurations.
```
Comment (plirglo) [inline docs/home/ARM.md]: Typo: "yet" instead "jet"

File: docs/home/ARM.md [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+name: default
+provider: any
+specification:
+  vault_enabled: true
+```
+
+### ```AWS``` provider
+
+- Important is to specify the right ```arm64``` machine type for component which can be found [here](https://aws.amazon.com/ec2/instance-types/a1/).
+- Important is to specify the right ```arm64``` OS image which currently is only ```CentOS 7.9.2009 aarch64```.
```
Comment (plirglo) [inline docs/home/ARM.md]: Maybe correct instead right? (just suggestion)

File: docs/home/ARM.md [hunk 5]
```diff
@@ -0,0 +1,568 @@
...
+
+## Support
+
+Below we give the current state of ```arm64``` support across the different providers, operating systems, components and applications. Make sure to check the ***notes*** for limitations that might still be present for supported components or applications.
+
+Besides making sure that the selected providers, operating systems, components and applications are supported with the tables below any other configuration for ```Epicli``` will work the same on ```arm64``` as they do on ```x86_64```. ```Epicli``` will return an error if any configuration is used that is not supported by the ```arm64``` architecture.
+
+### Providers
+
+| Providers | CentOS 7.x | RedHat 7.x | Ubuntu 18.04 |
```
Comment (plirglo) [inline docs/home/ARM.md]: I would rather use singular (Provider, Component, Application) in all tables, or at least keep them consistent

**Review by rafzei — DISMISSED** [review]
:rocket:

File: docs/home/ARM.md [hunk 6]
```diff
@@ -0,0 +1,569 @@
+# ARM
+
+From Epiphany v1.1.0 preliminary support for the ```arm64``` architecture was added. As the ```arm64``` architecture is relatively new to the datacenter at the time of writing only a subset of providers, operating system, components and applications are supported. Support will be extended in the future when there is a need for it.
```
Comment (plirglo) [inline docs/home/ARM.md]: operating system**s**

**Review by plirglo — APPROVED** [review]

**Review by rafzei — APPROVED** [review]

**Pull request merged**